<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="sessionStore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="maxCount">
     <number>10</number>
    </property>
    <item>
     <property name="text">
      <string>Session 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Session 10</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="earButtonStrong">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach to Earlobes (STRONG)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Session</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Session</string>
    </property>
   </widget>
   <widget class="QFrame" name="BatteryFrame">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="batteryLabel_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="batteryLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>BATTERY LEVEL: </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="SHLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Session History</string>
    </property>
   </widget>
   <widget class="QLabel" name="STLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Session Type </string>
    </property>
   </widget>
   <widget class="QLabel" name="SGLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Session Group</string>
    </property>
   </widget>
   <widget class="QSlider" name="UserDesSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="UserDesLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>User Designated Time</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SessionType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Delta (2.5-5Hz)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Theta (6-8Hz)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alpha (9-11Hz)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 Hz</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="PowerButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>POWER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="earButtonWeak">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach to Earlobes (WEAK)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detachEars">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Detach from Earlobes</string>
    </property>
   </widget>
   <widget class="QLabel" name="IntensityLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection Level/Intensity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SessionGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <item>
     <property name="text">
      <string>20 Minute</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>45 Minute</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="SessionType_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="powerFrame">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="powerLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>POWER: OFF</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CESModeLight">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CES Mode Light</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>BatteryFrame</zorder>
   <zorder>sessionStore</zorder>
   <zorder>upButton</zorder>
   <zorder>downButton</zorder>
   <zorder>earButtonStrong</zorder>
   <zorder>saveButton</zorder>
   <zorder>loadButton</zorder>
   <zorder>SHLabel</zorder>
   <zorder>STLabel</zorder>
   <zorder>SGLabel</zorder>
   <zorder>UserDesSlider</zorder>
   <zorder>UserDesLabel</zorder>
   <zorder>SessionType</zorder>
   <zorder>PowerButton</zorder>
   <zorder>earButtonWeak</zorder>
   <zorder>detachEars</zorder>
   <zorder>IntensityLabel</zorder>
   <zorder>StartButton</zorder>
   <zorder>SessionGroup</zorder>
   <zorder>SessionType_2</zorder>
   <zorder>powerFrame</zorder>
   <zorder>timeLabel</zorder>
   <zorder>CESModeLight</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
